core/src/main/java/org/mule/runtime/core/privileged/processor/chain/AbstractMessageProcessorChain.java

```diff
@@ -341,38 +340,20 @@ private ReactiveProcessor applyInterceptors(List<ReactiveInterceptor> intercepto
     interceptors.addAll(additionalInterceptors);
 
     // #4 Wrap execution, before processing strategy, on flow thread.
-    interceptors.add((processor, next) -> {
-      String processorPath;
-      if (processor instanceof Component && ((Component) processor).getLocation() != null) {
-        processorPath = ((Component) processor).getLocation().getLocation();
```

Comment:
by moving this logic out of this interceptor chain into the `ComponentMessageProcessor` class you're making this logic only available for SDK based component. What about hand crafted components like the `mule` and `ee` base language elements?

Reply:
reverted this part